chia/farmer/farmer.py

```diff
@@ -212,13 +212,15 @@ async def start_task():
     def _close(self):
         self._shut_down = True
 
-    async def _await_closed(self):
+    async def _await_closed(self, shutting_down: bool = True):
```

Comment:
This isn't strictly necessary, as the farmer doesn't seem to have the same stop/restart pattern that's employed by the wallet. Still, I made the same `shutting_down` change here to keep the code in sync (though it would be nice to have a non-duplicated code solution...)

Reply:
maybe at some point we will make some base class, but i think we can add that to the list of growing refactors.